src/browser/api/application.js

```diff
@@ -1102,7 +1102,7 @@ function createAppObj(uuid, opts, configUrl = '') {
         // save the original value of autoShow, but set it false so we can
         // show only after the DOMContentLoaded event to prevent the flash
         opts.toShowOnRun = eOpts['autoShow'];
-        eOpts.show = false;
+        eOpts.show = isChromePageUrl(opts.url); // no API injection for chrome pages
```

Comment:
Maybe this way would show a clearer intent?
```js
// save the original value of autoShow, but set it false so we can
// show only after the DOMContentLoaded event to prevent the flash.
// don't do this for chrome pages since the openfin api isn't injected
if (!isChromePageUrl(opts.url)) {
  opts.toShowOnRun = eOpts['autoShow'];
  eOpts.show = false;
}
```

Reply:
updated